aten/src/ATen/native/native_functions.yaml

```diff
@@ -7794,6 +7794,11 @@
   dispatch:
     CompositeExplicitAutograd: masked_scatter
 
+- func: masked_scatter_backward(Tensor grad_output, Tensor mask, SymInt[] sizes) -> Tensor
+  dispatch:
+    CPU: masked_scatter_backward_symint
+    CUDA: masked_scatter_backward_symint
```

Comment:
Why not CompositeexplicitAutograd? This implementation will work for any device.